src/Resources/app/storefront/src/scss/category-extend.scss

```diff
@@ -0,0 +1,42 @@
+.category_div_adjust {
+  min-width:290px;
+}
+.category_div_adjust label{
+  display:inline-block;
+  width: auto;
+}
+.subcats-show {
+  max-height:500.313rem;
+  min-height:1.3125rem;
+  transition: all 0.5s;
+}
+.up-arrow {
+  transform:rotate(180deg);
+  transition: all 0.3s;
+  height:30px;
+  width:35px;
+display: inline-block;
+}
```

Comment:
Also newlines should be added between SCSS statements. See the other SCSS files for reference.